src/libaktualizr/package_manager/packagemanagerfake.cc

```diff
@@ -49,6 +49,13 @@ data::InstallationResult PackageManagerFake::install(const Uptane::Target &targe
   return data::InstallationResult(data::ResultCode::Numeric::kOk, "Installing fake package was successful");
 }
 
+void PackageManagerFake::completeInstall() const {
+    if (config.force_reboot) {
+      LOG_INFO << "Emulate the system reboot in case of the fake package manager";
```

Comment:
That's nice that we also have this logic in the fake package manager!

Maybe the log message could be simpler: "Emulating a system reboot"

Reply:
OK. Will make it simpler.